src/Avalonia.Controls/Shapes/Arc.cs

```diff
@@ -0,0 +1,99 @@
+using System;
+using Avalonia.Media;
+
+namespace Avalonia.Controls.Shapes
+{
+    public class Arc : Shape
+    {
+        /// <summary>
+        /// Defines the <see cref="StartAngle"/> property.
+        /// </summary>
+        public static readonly StyledProperty<double> StartAngleProperty =
+            AvaloniaProperty.Register<Arc, double>(nameof(StartAngle), 0.0);
+
+        /// <summary>
+        /// Defines the <see cref="SweepAngle"/> property.
+        /// </summary>
+        public static readonly StyledProperty<double> SweepAngleProperty =
+            AvaloniaProperty.Register<Arc, double>(nameof(SweepAngle), 0.0);
+
+        static Arc()
+        {
+            StrokeThicknessProperty.OverrideDefaultValue<Arc>(1);
+            AffectsGeometry<Arc>(BoundsProperty, StrokeThicknessProperty, StartAngleProperty, SweepAngleProperty);
+        }
+
+        /// <summary>
+        /// Gets or sets the angle at which the arc starts, in degrees.
+        /// </summary>
+        public double StartAngle
+        {
+            get { return GetValue(StartAngleProperty); }
+            set { SetValue(StartAngleProperty, value); }
+        }
+
+        /// <summary>
+        /// Gets or sets the angle at which the arc ends relative to the start angle, in degrees.
+        /// </summary>
+        public double SweepAngle
```

Comment:
Old summary comments?

Reply:
I *think* I updated that one...though given you had to ask, I'm guessing I didn't do a very good job of it. So uhhh...how would you explain the sweep angle property?

Reply:
On second look I see how it could be correct, still a little confusing for me.

> how would you explain the sweep angle property?

What about: "Gets or sets the angle, in degrees, added to the <see cref="StartAngle"/> defining where the arc ends. A positive value is clockwise, negative is counter-clockwise."

Reply:
That sounds much better to me, yeah.